src/components/input.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react"
+import styled from "styled-components"
+import colors from "../assets/colors"
+import * as fonts from "../assets/fonts"
+import { block } from "./helpers"
+
+interface InputProps extends React.HTMLProps<HTMLInputElement> {
```

Comment:
So I actually ended up addressing another issue from the one you described 😅  but it's related to shared properties between the Input and TextArea. In terms of the Interfaces, yes we can extract shared properties into a base Interface. But let's address it in another PR.